parsers/src/test/scala/uk/gov/ons/addressIndex/parsers/postTokenizeTreatmentTest.scala

```diff
@@ -21,19 +18,16 @@ class postTokenizeTreatmentTest extends FlatSpec with Matchers {
     )
 
     // When
-    val actual = Tokens.postTokenizeTreatment(input)
+    val actual = Tokens.postTokenizeTreatmentBuildingName(input)
```

Comment:
You are now testing only the  BuildingName treatment, not the complete treatment. Is there a possibility that any of the other methods could change the tokens or their content leading to unwanted results?